src/App.js

```diff
@@ -1,12 +1,29 @@
 /* eslint-disable linebreak-style */
```

Comment:
- [ ] Kindly remove this eslint disable line and fix the issue in the code.

Reply:
but the issue is somehow not fixing